misc/zpipe_append.c

```diff
@@ -100,7 +100,7 @@ static int def(FILE *source, FILE *dest, int window_bits, int _flush,
 		return Z_ERRNO;
 
 	out = malloc(CHUNK_o);
-	if (in == NULL) {
+	if (out == NULL) {
```

Comment:
Please, could you please give special attention to this part? I am not sure if this is the intended. It just made sense, since this verification was duplicated.